lib/bdf/tests/Benchmark/BdfParserBench.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace PhpTui\Bdf\Tests\Benchmark;
+
+use RuntimeException;
+use PhpTui\BDF\BdfParser;
+use PhpBench\Attributes\Iterations;
+use PhpBench\Attributes\Revs;
+
+#[Iterations(10)]
+#[Revs(25)]
+final class BdfParserBench
+{
+    public function benchParseRealFont(): void
+    {
+        $contents = file_get_contents(__DIR__ . '/../../fonts/6x10.bdf');
+
+        if (false === $contents) {
+            throw new RuntimeException(
+                'Could not read file'
+            );
+        }
+
+        (new BdfParser())->parse($contents);
```

Comment:
it won't make much difference, but it's better to do any setup _outside_ of the `bench` method as it can muddy the results. The simplest way is to do it in the constructor (you can also add custom `setUp` methods but `__construct` is fine

Reply:
Oh, that's a really good advice. It makes all sense. I'll do it. =)

Reply:
```
1: <current> 2: main

Table
=====

<current>
+----------------+-----------------------+---------+----------+----------+----------+--------+-----------+
| benchmark      | subject               | memory  | min      | max      | mode     | rstdev | stdev     |
+----------------+-----------------------+---------+----------+----------+----------+--------+-----------+
| BdfParserBench | benchParseRealFont () | 6.031mb | 34.892ms | 36.568ms | 35.168ms | ±1.45% | 514.613μs |
+----------------+-----------------------+---------+----------+----------+----------+--------+-----------+

main
+----------------+---------------------------+---------+----------+----------+----------+--------+-----------+
| benchmark      | subject                   | memory  | min      | max      | mode     | rstdev | stdev     |
+----------------+---------------------------+---------+----------+----------+----------+--------+-----------+
| BdfParserBench | benchParseRealFont ()     | 6.031mb | 38.240ms | 41.477ms | 39.279ms | ±2.42% | 960.798μs |
+----------------+---------------------------+---------+----------+----------+----------+--------+-----------+
```